arlas-tests/src/test/java/io/arlas/server/rest/explore/AbstractFilteredTest.java

```diff
@@ -574,12 +574,7 @@ public void testInvalidFilterParameters() throws Exception {
         handleInvalidParameters(header(request.filter));
         request.filter.f = null;
 
-        //Q
-        request.filter.q = Arrays.asList(new MultiValueFilter<>("fullname:My:name"));
-        handleInvalidParameters(post(request));
-        handleInvalidParameters(get("q", request.filter.q.get(0).get(0)));
-        handleInvalidParameters(header(request.filter));
-        request.filter.q = null;
+
```

Comment:
It would be good to replace the invalid parameters test by a handleMatchingQueryFilter test